### PR TITLE
fix(web): allow both scroll right and down

### PR DESCRIPTION
**Pull request by antonymarion**
ref: MANAGER-12658

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/view-dns-history-MANAGER-12199`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-12658
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)

## Description

Add both scroll right and down for big dns zone texts

### CODE REVIEW

File: packages/manager/apps/web/client/app/dns-zone/history/dashboard/dashboard.html [hunk 1]
```diff
@@ -126,13 +126,13 @@
                     data-on-dismiss="$ctrl.closeVisualizeDnsPopup()"
                 >
                     <oui-spinner ng-if="$ctrl.loadingDnsZoneData"></oui-spinner>
-                    <div
-                        style="white-space: pre-wrap;"
+                    <p
+                        style="white-space: pre-wrap;overflow: auto;max-height: 50vh"
```
Comment (darsene) [inline packages/manager/apps/web/client/app/dns-zone/history/dashboard/dashboard.html:130]: you should define a css class and merge it with `class="mb-0"`

File: packages/manager/apps/web/client/app/dns-zone/history/dashboard/dashboard.html [hunk 2]
```diff
@@ -126,13 +126,13 @@
...
                         ng-if="!$ctrl.loadingDnsZoneData"
                         class="mb-0"
                     >
                         {{ $ctrl.dnsZoneData }}
```
Comment (darsene) [inline packages/manager/apps/web/client/app/dns-zone/history/dashboard/dashboard.html:134]: use `ng-bind-html` instead

File: packages/manager/apps/web/client/app/dns-zone/history/dashboard/dashboard.html (hunk 1 above)
Comment (qpavy) [inline packages/manager/apps/web/client/app/dns-zone/history/dashboard/dashboard.html:130]: use max-height: 50dvh or 50svh would be better for mobile accessibility

Comment (antonymarion) [reply packages/manager/apps/web/client/app/dns-zone/history/dashboard/dashboard.html:130]: Thks I did not know this one :)

**Review by darsene — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

**Pull request merged**